tests/comparePurchaseUrgency.test.js

```diff
@@ -0,0 +1,7 @@
+// import { comparePurchaseUrgency } from '../src/api';
+
+describe('Compare Purchase Urgency', () => {
+	test('it works', () => {
+		expect(true).toBe(true);
+	});
+});
```

Comment:
This seems like a placeholder. Could we remove it for now and add a meaningful test for the `comparePurchaseUrgency` function once it's ready?

Reply:
You are right, it was very much a prototype, I will work on creating proper tests